backend/.env

```diff
@@ -0,0 +1,9 @@
+DB_DOMAIN = "loxotron_backend_db"
```

Comment:
Dude, It's an Open Source project whole internet sees your credentials. Next time please use at least git crypt or set up local MongoDB.